doc/man/scons.xml

```diff
@@ -1022,6 +1022,11 @@ needing the <filename>.py</filename> extension.
 The names &SConstruct; and &SConscript; are now
 recognized without requiring
 <filename>.py</filename> suffix.
+<para>
+</para>
+<emphasis>Changed in version 4.7.1</emphasis>:
+The name &SCsub; is now recognized without requiring
```

Comment:
To reference it this way, would have to define the entity in `doc/scons.mod`.  I think it's okay to just spell it out `<filename>SCsub</filename>`.